source/includes/read-write-pref.scala

```diff
@@ -0,0 +1,80 @@
+import org.mongodb.scala._
+import com.mongodb.{ TransactionOptions, ReadConcern, ReadPreference, WriteConcern }
+import scala.jdk.CollectionConverters._
+
+object ReadWritePref {
+
+  def main(args: Array[String]): Unit = {
+
+    // Uses the settings builder methods to set read and write settings for the client
+    // start-client-settings
+    val mongoClient = MongoClient(MongoClientSettings.builder()
+        .applyConnectionString(ConnectionString("mongodb://localhost:27017/"))
+        .readPreference(ReadPreference.secondary())
+        .readConcern(ReadConcern.LOCAL)
+        .writeConcern(WriteConcern.W2)
+        .build())
+    // end-client-settings
+
+    // Uses connection URI parameters to read and write settings for the client
+    // start-client-settings-uri
+    val uriClient = MongoClient("mongodb://localhost:27017/?readPreference=secondary&w=2&readConcernLevel=local")
+    // end-client-settings-uri
+
+    // Sets read and write settings for the transaction
+    // start-transaction-settings
```

Comment:
Note to tech reviewer: this section of the file won't compile, since there's no clientSession

Reply:
Any reason not to create one?

Reply:
I thought it might overcomplicate the example but it's only a couple extra lines, so added!